actions/generate-changelog/action.yaml

```diff
@@ -0,0 +1,62 @@
+name: "Generate changelog"
+description: "Generate changelog after release"
+inputs:
+  # required inputs
+  changelog-file:
+    description: Path to the Changelog.md file
+    required: true
+  github-token:
+    description: "The GitHub token for committing the changes."
+    required: true
+  new-tag:
+    description: Version after bump
+    required: true
+  old-tag:
+    description: Previous version
+    required: true
+
+  # unrequired inputs
+  config:
+    description: Path to the changelog config JSON file
+    required: false
+    default: "./.github/changelog-config.json"
+  output:
+    description: "Name of the output file for the generate changelog step"
+    required: false
+    default: "changes.md"
+  fetchReviewers:
+    description: "Will enable fetching the users/reviewers who approved the PR"
+    required: false
+    default: "false"
+  fetchReleaseInformation:
+    description: "Will enable fetching additional release information from tags."
+    required: false
+    default: "false"
+  commitMode:
+    description: "Special configuration for projects which work without PRs."
+    required: false
+    default: "false"
+
+runs:
+  using: "composite"
+  steps:
+    - name: "Create changelog"
+      id: build_changelog
+      uses: mikepenz/release-changelog-builder-action@v3.7.0
+      with:
+        token: "${{ inputs.github-token }}"
+        configuration: "${{ inputs.config }}"
+        fromTag: "${{ inputs.old-tag }}"
+        toTag: "${{ inputs.new-tag }}"
+        outputFile: "${{ inputs.output }}"
```

Comment:
It seems like this action has a number of output, among others `outputs.changelog`. Can we also use an output instead of writing it to a temporary file?

Reply:
we could use it but the generated changelog is a kind of standalone changelog, because it just generate changelogs for the current tag, nbot for the previous..

Reply:
Yes, the temporary file was only a workaround. I think using the output is much cleaner because we have one less parameter and users don't need to worry that we add a file that they might accidentally commit and push

Reply:
if we do not want to modify the `Changelog` file in this composite action but want to return the raw `Changelog` instead, then we do not need a composite action at all. because the used action already does that.

Reply:
@MichaelKora @yannick-roeder  Just to clarify, I do want to update the changelog file, but I also need the generated changelog for the GitHub release later on, for example in Quick:
- Changelog update: https://github.com/bakdata/quick/blob/master/CHANGELOG.md#080-2022-10-14
- Release note: https://github.com/bakdata/quick/releases/tag/0.8.0